[2022-08-07 03:49:27,491] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.final_dag scheduled__2022-08-06T00:00:00+00:00 [queued]>
[2022-08-07 03:49:27,500] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.final_dag scheduled__2022-08-06T00:00:00+00:00 [queued]>
[2022-08-07 03:49:27,501] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-07 03:49:27,501] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-07 03:49:27,502] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-07 03:49:27,517] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): final_dag> on 2022-08-06 00:00:00+00:00
[2022-08-07 03:49:27,523] {standard_task_runner.py:52} INFO - Started process 19784 to run task
[2022-08-07 03:49:27,525] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'final_dag', 'scheduled__2022-08-06T00:00:00+00:00', '--job-id', '141', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmp96ug66q0', '--error-file', '/tmp/tmpjgdab35c']
[2022-08-07 03:49:27,527] {standard_task_runner.py:80} INFO - Job 141: Subtask final_dag
[2022-08-07 03:49:27,593] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.final_dag scheduled__2022-08-06T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-07 03:49:27,667] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=final_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-08-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-06T00:00:00+00:00
[2022-08-07 03:49:27,746] {python.py:173} INFO - Done. Returned value was: None
[2022-08-07 03:49:27,760] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=final_dag, execution_date=20220806T000000, start_date=20220807T034927, end_date=20220807T034927
[2022-08-07 03:49:27,778] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-07 03:49:27,809] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
